docs/docs/concepts/features/external-user-grant.md

```diff
@@ -0,0 +1,40 @@
+---
+title: External user grant
```

Comment:
```suggestion
title: External User Grant
```

Reply:
I guess we use title case, right?

Reply:
No, we use lower case as per contributing guide :)

Reply:
I guess you meant the google guidelines which use sentence case, right.

https://developers.google.com/style/capitalization

Reply:
Yes.
Side-note: title case is easy to achieve with css. The other way round not if we use own names (eg, ZITADEL, Organization, ...)